docker-compose.yml

```
@@ -384,6 +395,21 @@ services:
         <<: *depends_on-healthy
       web:
         <<: *depends_on-healthy
+  vroom:
+    <<: *restart_policy
+    image: "$VROOM_IMAGE"
+    environment:
+      SENTRY_KAFKA_BROKERS_PROFILING: "kafka:9092"
+      SENTRY_KAFKA_BROKERS_OCCURRENCES: "kafka:9092"
+      SENTRY_BUCKET_PROFILES: file://localhost//var/lib/sentry-profiles
```

I'd prefer:

```suggestion
      SENTRY_BUCKET_PROFILES: "${SENTRY_BUCKET_PROFILES:-file://localhost//var/lib/sentry-profiles}"
```

That way, people can adjust their vroom storage backend config easier (ie. they can use GCS or S3 or Azure Blob), defined from .env or .env.custom without having them to change docker-compose.yml

---

Or, you might prefer `VROOM_BUCKET_PROFILES`.

---

I'd discourage this as it might be better to override the compose file with `docker-compose.override.yml` until we get a dedicated `vroom` config file.

---

Okay, agreed.